app/src/lib/parser/pairings.js

```diff
@@ -0,0 +1,28 @@
+import { REGEXES } from "$lib/regexes"
```

Comment:
`pnpm precommit` needs to be run on this file... I see formatting issues as well as linter issues.